test/fuzzer/RyanJsonFuzzer.c

```
@@ -0,0 +1,688 @@
+
+	// 按 index 删除
+	uint32_t idx = RyanJsonGetSize(pJson) % size;
+	RyanJsonDeleteByIndex(pJson, (size % 25) ? idx : 0);
+
+	return RyanJsonTrue;
+}
+
+static RyanJsonBool_e RyanJsonFuzzerTestByMinify(const char *data, uint32_t size)
+{
+	char *buf = malloc(size + 100);
+	memset(buf, 0, size + 100);
```

Cppcheck (2.18.0)

[warning] If memory allocation fails, then there is a possible null pointer dereference

(nullPointerOutOfMemory)